src/main/drivers/pwm_output.c

```diff
@@ -141,7 +141,13 @@ static void pwmWriteOneshot(uint8_t index, uint16_t value)
 
 static void pwmWriteMultiShot(uint8_t index, uint16_t value)
 {
-    *motors[index]->ccr = 60001 * (value - 1000) / 250000 + 60;
+
+#if defined(STM32F40_41xxx) || defined(STM32F411xE)
+    *motors[index]->ccr = (uint16_t)((float)(value-1000) / 1.04166f)+ 240;
```

Comment:
I believe float division is faster than int division on the STM32F4. From what I understand, unless we're using bit shifting of simple values it's better to multiply and divide floats than ints. I'll run a quick test and find out.